stellargraph/datasets/datasets.py

```diff
@@ -707,3 +707,42 @@ def load(self):
             ``graph``.
         """
         return _load_tsv_knowledge_graph(self)
+
+
+class IAEnronEmployees(
+    DatasetLoader,
+    name="IAEnronEmployees",
+    directory_name="ia-enron-employees",
+    url="http://nrvis.com/download/data/dynamic/ia-enron-employees.zip",
+    url_archive_format="zip",
+    expected_files=["ia-enron-employees.edges", "readme.html"],
+    description="A dataset of edges that represent emails sent from one employee to another."
+    "There are 50572 edges, and each of them contains timestamp information. "
+    "Edges refer to 151 unique node IDs in total.",
+    source="http://networkrepository.com/ia-enron-employees.php",
+    url_contains_directory=False,
+):
+    def load(self):
+        """
+        Load this data into a set of nodes and edges
+
+        Returns:
+            A tuple ``(nodes, edges)`` where each element is a Pandas DataFrame. Edges are kept
+            sorted in ascending order of time as per the original dataset.
+        """
+        self.download()
+
+        edges_path = self._resolve_path("ia-enron-employees.edges")
+        edges = pd.read_csv(edges_path, sep=" ", header=None)
+
+        # clean up columns
+        edges.columns = ["source", "target", "x", "time"]
+        edges[["source", "target"]] = edges[["source", "target"]].astype(str)
+        edges = edges.drop(columns=["x"])  # unused column
```

Comment:
Do you know what this column is meant to be? It seems to always be 1?

Reply:
Network repository seems to only describe the graph properties of the data, I couldn't find explanation for any of the columns, and there aren't any column names in the dataset. Maybe it was a column relevant for a larger version of the dataset? But as you say, it always seems to be 1 here.